src/site/includes/header.html

```diff
@@ -116,5 +116,3 @@ <h3 class="usa-alert-heading">{{ banner.title }}</h3>
     </div>
   </div>
 {% endif %}
-
-<script async src="/js/usa-search.js"></script>
```

Comment:
As part of cleaning up, can you remove this file from `src/site/assets/js`?

Reply:
deleted!